detty_server/migrations/0001_initial.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.3 on 2023-07-04 22:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('user_id', models.UUIDField()),
```

Comment:
Change id to user_id, remove user_id and make user_id a UUID field
```suggestion
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('user_id', models.UUIDField()),
```